automation_tools/scripts/ga-migration/utils.py

```diff
@@ -281,3 +281,23 @@ def read_yaml(filepath):
                 print(exc)
     else:
         logging.info("SKIPPED TASK. No %s found" % filepath)
+
+
+def add_nl_at_eof(filepath):
```

Comment:
Minor: could we have a less cryptic function name. At first sight, it's not very clear what it does. Prefer lengthy names rather than short and cryptic.